src/bits.rs

```diff
@@ -155,13 +155,24 @@ impl<const BITS: usize, const LIMBS: usize> Uint<BITS, LIMBS> {
     #[inline]
     #[must_use]
     pub const fn leading_zeros(&self) -> usize {
+        let fixed = Self::MASK.leading_zeros() as usize;
```

Comment:
this could be a computed const?

Reply:
it doesn't matter for perf in this context, it does matter for potential reuse by other fns or downstream users. the question is whether it's more intuitive and legible to access this as `Self::MASK.leading_zeros()` or as `Self::MASK_LEADING_ZEROS`. I'm not convinced it's better either way (especailly as `MASK` is not exactly a clear constant name on its own)